airflow/models/dag.py

```diff
@@ -1842,16 +1842,17 @@ def deactivate_stale_dags(expiration_date, session=None):
         :type expiration_date: datetime
         :return: None
         """
-        for dag in session.query(
-                DagModel).filter(DagModel.last_scheduler_run < expiration_date,
-                                 DagModel.is_active).all():
+        query = session.query(DagModel).filter(DagModel.last_scheduler_run < expiration_date,
```

Comment:
Oooh crap, I don't update this column anymore on master, so I don't think this is right. But there also isn't any global setting I do update anymore.

Even SerialzedDag.updated_at might not be right, as if the dag hasn't changed, that value won't be updated anymore.

Reply:
Ooh is it? Then should it be taken care of while doing serialization itself? Like, mark the dag inactive if it doesn't find it anymore?